lib/jna/clink_jna.cc

```diff
@@ -106,21 +117,25 @@ void SparseVector_delete(SparseVectorJNA *vector) { delete vector; }
 
 SparseVectorJNA *OneHotEncoderModel_transform(clink::OneHotEncoderModel *model,
                                               const int value,
-                                              const int columnIndex) {
-  auto sparse_vector = model->transform(value, columnIndex);
-  CLINK_JNA_HANDLE_ERROR(sparse_vector.takeError())
+                                              const int column_index) {
+  auto sparse_vector = model->transform(value, column_index);
+  CLINK_JNA_HANDLE_ASYNC_ERROR(sparse_vector)
+  if (sparse_vector.IsError()) {
+    return NULL;
+  }
   return new SparseVectorJNA(sparse_vector.get());
 }
 
 clink::OneHotEncoderModel *
```

Comment:
The position of whitespace to pointers is decided by clang-format. TFRT's clang-format style is google, but so far by now we have been using `-style=llvm`. I guess we should also change it to google style since this is how we have stated in README.

clang-format has two properties that are related to this issue. 
- `PointerAlignment`: How to align pointers. If its value is right, then we get `OneHotEncoderModel *`. If it's left, then we have `OneHotEncoderModel*`.
- `DerivePointerAlignment`: Whether to apply the mostly used pointer alignment cases to all. It's value could be true or false. This property seems to have a higher property than `PointerAlignment`, which means if there are more `OneHotEncoderModel *` than `OneHotEncoderModel*`, then even if we set `PointerAlignment:Left`, it will still change `OneHotEncoderModel*` to `OneHotEncoderModel *`.

In Google style the values are as below:
```yaml
DerivePointerAlignment: true
PointerAlignment: Left
```
In LLVM style:
```yaml
DerivePointerAlignment: false
PointerAlignment: Right
```
If we apply Google style, we cannot solve the `OneHotEncoderModel *` problem unless we make additional manual adjustments. In fact what we want in this discussion is 
```yaml
DerivePointerAlignment: false
PointerAlignment: Left
```
What do you think we should choose in this case?

Reply:
Per discussion offline, let's pick one from the Google and llvm style that is closer to TFRT style.